apps/db/supabase/migrations/20250818153838_add_rsvp.sql

```diff
@@ -0,0 +1,289 @@
+create type "public"."event_attendee_status" as enum ('pending', 'accepted', 'declined', 'tentative');
+
+create type "public"."event_status" as enum ('active', 'cancelled', 'completed', 'draft', 'confirmed');
+
+create table "public"."event_attendees" (
+    "id" uuid not null default gen_random_uuid(),
+    "event_id" uuid not null,
+    "user_id" uuid,
+    "status" event_attendee_status default 'pending'::event_attendee_status,
+    "response_at" timestamp with time zone,
+    "created_at" timestamp with time zone not null default now(),
+    "updated_at" timestamp with time zone default now()
+);
+
+
+alter table "public"."event_attendees" enable row level security;
+
+create table "public"."event_invitations" (
+    "id" uuid not null default gen_random_uuid(),
+    "event_id" uuid,
+    "invitee_id" uuid,
+    "invitation_token" uuid default gen_random_uuid(),
+    "created_at" timestamp with time zone not null default now()
+);
+
+
+alter table "public"."event_invitations" enable row level security;
+
+create table "public"."workspace_scheduled_events" (
+    "id" uuid not null default gen_random_uuid(),
+    "ws_id" uuid,
+    "title" text not null,
+    "description" text,
+    "start_at" timestamp with time zone,
+    "end_at" timestamp with time zone,
+    "location" text,
+    "color" text not null default 'BLUE'::text,
+    "creator_id" uuid,
+    "is_all_day" boolean default false,
+    "requires_confirmation" boolean default true,
+    "status" event_status default 'active'::event_status,
+    "created_at" timestamp with time zone not null default now(),
+    "updated_at" timestamp with time zone default now()
+);
```

Comment:
We already have the `workspace_calendar_events` table, is there a good reason why we're creating a new table here instead of utilizing / extending the old one?